src/sentry/static/sentry/app/views/settings/account/accountIdentities.tsx

```diff
@@ -1,31 +1,40 @@
 import React from 'react';
+import {RouteComponentProps} from 'react-router/lib/Router';
 import styled from '@emotion/styled';
 
 import {disconnectIdentity} from 'app/actionCreators/account';
 import {t} from 'app/locale';
 import AsyncView from 'app/views/asyncView';
 import Button from 'app/components/button';
+import {Identity} from 'app/types';
 import EmptyMessage from 'app/views/settings/components/emptyMessage';
 import {Panel, PanelBody, PanelHeader, PanelItem} from 'app/components/panels';
 import SettingsPageHeader from 'app/views/settings/components/settingsPageHeader';
 
 const ENDPOINT = '/users/me/social-identities/';
 
-class AccountIdentities extends AsyncView {
-  getEndpoints() {
-    return [['identities', ENDPOINT]];
-  }
+type Props = RouteComponentProps<{}, {}>;
 
-  getTitle() {
-    return 'Identities';
-  }
+type State = {
+  identities: Identity[];
```

Comment:
```suggestion
  identities: Identity[] | null;
```

AsyncView sets it to null in case of error 😉